app/assets/v2/js/pages/bounty_details.js

```diff
@@ -834,11 +834,9 @@ const build_uri_for_pull_bounty_from_api = function() {
 };
 
 var pull_bounty_from_api = function() {
-  let all_results = [];
-
   $.get(build_uri_for_pull_bounty_from_api()).then(results => {
-    all_results = sanitizeAPIResults(results);
-    return all_results;
+    // do not sanitize issue_description. showdown will handle that.
+    return sanitizeAPIResults(results, 'issue_description');
```

Comment:
have you tested that it is not possible to inject `script`s into the bounty page after doing this?

Reply:
It did just fine with the data from the API.
However you a are right! I tested several inputs and injection is possible.
I will come up with a solution